README.md

```diff
@@ -2,20 +2,44 @@
 
 [![Go Reference](https://pkg.go.dev/badge/mvdan.cc/gofumpt/format.svg)](https://pkg.go.dev/mvdan.cc/gofumpt/format)
 
-	go install mvdan.cc/gofumpt@latest
+`gofumpt` is a code formatter for Go that enforces rules that are stricter than
+those used by `gofmt`. That is, any Go code formatted with `gofumpt` will not be
+modified when formatted by `gofmt`.
 
-Enforce a stricter format than `gofmt`, while being backwards compatible.
-That is, `gofumpt` is happy with a subset of the formats that `gofmt` is happy with.
+- [Introduction](#introduction)
+- [Added rules](#added-rules)
+  - [Extra rules behind `-extra`](#extra-rules-behind--extra)
+- [Installation](#installation)
+- [Configuring your editor for use with `gofumpt`](#configuring-your-editor-for-use-with-gofumpt)
+  - [Visual Studio Code](#visual-studio-code)
+  - [GoLand](#goland)
+  - [Vim](#vim)
+  - [Neovim](#neovim)
+  - [Emacs](#emacs)
+  - [Sublime Text](#sublime-text)
+- [Roadmap](#roadmap)
+- [Frequently Asked Questions](#frequently-asked-questions)
+- [Contributing](#contributing)
+- [License](#license)
```

Comment:
I'm conflicted about an index. On one hand, it's nice to be able to jump around. On the other, the current README jumps straight to the information, whereas the new one fills nearly an entire page with just the index. Overall I think this is not a good idea - if the README is so long that it can't possibly be navigated directly, then we've done something wrong.

What probably got you in this direction is the fact that the "added rules" section is so long. I've wanted to make it vertically shorter for some time, such as by using an HTML table. I'd rather go in that direction.